### PR TITLE
[FIX] Socket should also connect to redis specified in .env

### PR DESCRIPTION
**Pull request by pavankumarkatakam**
**In raising this pull request, I confirm the following (please check boxes):**

- [*] I have read and understood the [contributing guidelines](/.github/CONTRIBUTING.md)?
- [*] I have checked that another pull request for this purpose does not exist.
- [*] I have considered, and confirmed that this submission will be valuable to others.
- [*] Do the PHPCI tests pass?
- [*] Does the StyleCI test pass?

_NOTE: The last 2 are not required to open a PR and can be done afterwards /
while the PR is open._

---

### Description of change

*Please provide a description of the change here.*

Im actually using the redis in docker means another machine it has its own ip address, all are working fine but notifications when deploying are not working, when i debugged in the `socket.js` it is connecting to local redis instance rather than redis that is specified in the .env file. I think it is better to connect to redis which is specified in .env. Generally when using the remote redis it is preferrable to have a password so i have given the option to specify the `redis_password`

### CODE REVIEW

**Comment by REBELinBLUE** [comment]
Thanks, I'll merge it this evening

**Comment by REBELinBLUE** [comment]
Could you add the default values to `.env.example` as well?

Thanks

File: socket.js [hunk 1]
```diff
@@ -7,7 +7,10 @@ var debug = (process.env.APP_DEBUG === 'true' || process.env.APP_DEBUG === true)
 
 var Redis = require('ioredis');
 var redis = new Redis({
-    db: process.env.REDIS_DATBASE || 0
+    port: process.env.REDIS_PORT || 6379, 
+    host: process.env.REDIS_HOST || '127.0.0.1', 
+    db: process.env.REDIS_DATBASE || 0,
```
Comment (REBELinBLUE) [inline socket.js]: Just noticed, this should be `REDIS_DATABASE` (I know, it was wrong before also). Would you mind changing it?

Thanks

**Comment by pavankumarkatakam** [comment]
Default values is nothing for password and database, so im leaving it empty.

**Comment by REBELinBLUE** [comment]
Thanks

**Comment by REBELinBLUE** [comment]
Turns out that doesn't work, leaving them empty results in them becoming empty values in the config, not the defaults so I have set the defaults in `.env.example`

**Pull request merged**